### DISCUSSION TITLE
Ansible clutters the audit log

### DISCUSSION THREAD

**Discussion by re-mscho** [gravitational/teleport#6574] [Q&A]
Hallo,
I've just configured teleport and ansible to work together to manage a node connected by a reverse tunnel which is great.
When I now view the audit logs, there are many entries per ansible playbook run which I somehow expected.

Is is possible to configure teleport and/or ansible to not clutter the audit log that much?

Thanks in advance

**Comment by Valien** [comment]
Hi there @re-mscho! Saw your post in the Slack channel so figured to respond here as well. Ansible by its nature is a very verbose logging system (if you ever want to see what's happening add a `-v` or `-vvvv`). If you are using Tower all that data is captured for audit/logging. If you are using CLI only I do not believe there is a way to suppress this info.

So Teleport is just intercepting all this data and logging it for its audit trails. Now you could potentially tweak the `teleport.yaml` file on your auth host (if you are self-hosted) or on the nodes and change the `log` settings. By default it is set for `INFO`.

**Comment by webvictim** [comment]
You can add this in your `ansible.cfg` to reduce the volume of log spam:

```
[ssh_connection]
use_tty = no
```

**Reply by re-mscho** [reply]
Thank you, I've already done that as suggested by Jeff Anderson on Slack.
But as he mentioned, this does not prevent the audit log from being filled with event entries.
